aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -972,7 +972,7 @@ static dispatch_data_t getSectionData(const std::string& name) {
 
 void MetalShaderLibrary::exec_unary_kernel(TensorIteratorBase& iter,
                                            const std::string& name,
-                                           std::optional<int64_t> extra) {
+                                           c10::optional<int64_t> extra) {
```

Comment:
wait why then add these back?

Reply:
See the test plan: will remove this before final merge :)